client/state/selectors/get-jetpack-recommendations-url.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { getUrlParts, getUrlFromParts } from 'calypso/lib/url';
+
+export default function getJetpackRecommendationsUrl( state ) {
+	const site = getSelectedSite( state );
+
+	const adminUrl = get( site, 'options.admin_url' );
```

Comment:
As lodash is being slowly removed from calypso, I think it might be a good idea to use pure JS here instead of `get`. How about this?

```suggestion
	const adminUrl = site?.options?.admin_url;
```

Reply:
Good idea, I made it pure JS and took the lodash import out.